Juniper/MX-series/telemetry_dialin.conf

```diff
@@ -1,3 +1,29 @@
+/* Create a local user for gnmi-user per Juniper docs
+(https://www.juniper.net/documentation/us/en/software/junos/grpc-network-services/topics/topic-map/grpc-services-configuring.html#task-configure-gnoi-user-account)
+*/
+
+/* The allow-commands-regexps and deny-commands-regexps statements were introduced in Junos OS Release 18.1
+ssl statement introduced in Junos OS Release 16.1
+mutual-authentication, client-certificate-request, and certificate-authority options added in Junos OS Release 17.4R1
+hot-reloading option added in Junos OS Release 20.4R1
+use-pki option added in Junos OS Evolved Release 22.2R1
+*/
+
+system {
+    login {
+        class {
+            grpc-operator {
+                permissions all;
```

Comment:
after some vmx testing, it looks like permissions "view" will be sufficient vs "all" to reduce new user perms.